docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -209,6 +209,7 @@ For example, on Command Prompt use:
 +
 ----
 C:\> set HYPERV_VIRTUAL_SWITCH=External (Wireless)
+C:\> setx HYPERV_VIRTUAL_SWITCH %HYPERV_VIRTUAL_SWITCH%
```

Comment:
I think this will persist the setting and we should add message like
```
If you want to persist this environment variable, perform as below:
```

something on this line.